timesketch/lib/charts/manager.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This file contains a class for managing charts."""
+
+from __future__ import unicode_literals
+
+
+class ChartManager(object):
+    """The chart manager."""
+
+    _class_registry = {}
+
+    @classmethod
+    def get_charts(cls):
+        """Retrieves the registered charts.
+
+        Yields:
+            tuple: containing:
+                unicode: the uniquely identifying name of the chart
+                type: the chart class.
+        """
+        for agg_name, agg_class in iter(cls._class_registry.items()):
+            yield agg_name, agg_class
+
+    @classmethod
+    def get_chart(cls, chart_name):
+        """Retrieves a class object of a specific chart.
+
+        Args:
+            chart_name (unicode): name of the chart to retrieve.
+
+        Returns:
+            Chart class object.
+        """
+        return cls._class_registry[chart_name.lower()]
+
+    @classmethod
+    def register_chart(cls, chart_class):
+        """Registers an chart class.
```

Comment:
s/an/a/

as in Registers a chart class.

Reply:
Done.